eZ/Publish/Core/REST/Server/Service/RootResourceBuilder.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * File containing the resource service class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ *
+ * @version //autogentag//
+ */
+namespace eZ\Publish\Core\REST\Server\Service;
+
+use eZ\Publish\Core\REST\Common\Values;
+use eZ\Publish\Core\REST\Common\Values\Root;
+use Symfony\Component\ExpressionLanguage\ExpressionLanguage;
+use Symfony\Component\Routing\RouterInterface;
+
+class RootResourceBuilder
```

Comment:
I'd make `RootResourceBuilder` an interface, and rename this implementation to something that matches what it does (expression language + router).